tests/s25Main/lua/testLua.cpp

```diff
@@ -588,6 +588,7 @@ BOOST_AUTO_TEST_CASE(World)
     BOOST_TEST(obj2->GetItemID() == 501u);
     BOOST_TEST(obj2->GetItemFile() == 0xFFFFu);
     BOOST_TEST(obj2->GetSize() == 1u);
+    BOOST_TEST(obj2->IsAnimated() == false);
```

Comment:
Please don't use `== false` or `== true` but rather e.g.:
```suggestion
    BOOST_TEST(!obj2->IsAnimated());
```